README.adoc

```diff
@@ -160,7 +160,7 @@ It includes many annotations that can automatically configure a slice of your ap
 
 === spring-boot-loader
 Spring Boot Loader provides the secret sauce that allows you to build a single jar file that can be launched using `java -jar`.
-Generally you will not need to use `spring-boot-loader` directly, but instead work with the link:spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin[Gradle] or link:spring-boot-project/spring-boot-tools/spring-boot-maven-plugin[Maven] plugin.
+Generally, you will not need to use `spring-boot-loader` directly but instead, work with the link:spring-boot-project/spring-boot-tools/spring-boot-gradle-plugin[Gradle] or link:spring-boot-project/spring-boot-tools/spring-boot-maven-plugin[Maven] plugin.
```

Comment:
Should we have this comma after "instead"?

Reply:
As "instead" is an introductory phrase, it is always followed by a comma. That's why I added a comma in this sentence.

Reply:
If instead were an introductory adverb it would be at the start of the sentence and, in that case, it should then be followed by a comma. As used here, I disagree that a comma is necessary. I think it would be better to reword the sentence to avoid using instead in its middle:

> Generally, you will not use `spring-boot-loader` directly but work with the Gradle or Maven plugin instead.

Reply:
Ok removing "instead" as directed.

Reply:
Placed "instead" at the end of the sentence.